File: exercise01-A/src/Program.cs
// Exercise 01-A: Line Totals

using System;
using System.Linq;

class Program
{
    static void Main()
    {
        string line;
        while ((line = Console.ReadLine()) != null)
        {
            Console.WriteLine(line.Trim().Split(' ').Select(int.Parse).ToArray().Sum());
        }
    }
}

File: exercise01-A/src/Program.java
import java.util.Scanner;

public class Program {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        while (scanner.hasNextLine()) {
            String line = scanner.nextLine();

            int sum = 0;
            for (String word : line.split(" ")) {
                sum += Integer.parseInt(word);
            }

            System.out.println(sum);
        }

        scanner.close();
    }
}
